### DISCUSSION TITLE
h.set is not a function

### DISCUSSION THREAD

**Discussion by wajidkhanp** [postmanlabs/postman-flows#231] [Defect]
I am having a simple flow where I am creating a list of data and using forEach on this list. 
I assign data from list to a variable string of my API 
when I run the flow, I see this  `h.set is not a function` in the error and my API is not getting executed. 
if I remove variable assignment in my API then I see my API getting executed number of items in the list but failing as variable in the API is not assigned with a value. 

https://user-images.githubusercontent.com/11093819/151090020-a251e308-f31d-460a-a7b0-ada16ded8b93.mov

Is this a bug or am I using flow constructs incorrectly ?

**Comment by saswatds** [comment]
Hey, @wajidkhanp are you trying to do the variable assignment using the `Scripts`? Would you be able to share the logs with us at help@postman.com ?

**Comment by wajidkhanp** [comment]
I am not assigning variables using the `Scripts` . Also, when I assign anything in variable I do not get to see any logs on console. 
Please advice how and where should I check the logs and i will be happy to share the same.  

<img width="1192" alt="Screen Shot 2022-01-25 at 7 54 31 PM" src="https://user-images.githubusercontent.com/11093819/151096388-7710744a-fa2f-4782-9f25-dfe5693a8eed.png">

**Reply by saswatds** [reply]
Hey @wajidkhanp, you can get the logs from `View > Developer > View Logs`. Meanwhile, let me try to debug this on my end.

**Reply by saswatds** [reply]
We have been able to find the issue and hot-fixing it immediately.

**Comment by wajidkhanp** [comment]
This issue is resolved now and we are good to close this.
